ui/src/alerting/components/EditCheckEO.tsx

```diff
@@ -33,9 +32,9 @@ import {
 
 interface DispatchProps {
   updateCheck: typeof updateCheck
-  setCurrentCheck: typeof setCurrentCheck
-  getCurrentCheck: typeof getCurrentCheck
-  updateCurrentCheck: typeof updateCurrentCheck
+  setTimeMachineCheck: typeof setTimeMachineCheck
```

Comment:
We used to have a linting rule that would prevent us from shadowing names in the same scope. Not a big deal but I found it helpful to distinguish between a function of type `T` versus one of type `(dispatch: Dispatch) => T` using an `on` prefix for the connected one (`onSetTimeMachineCheck`).

I don't think the `onSetTimeMachineCheck` style needs to be the one true way, just thought I would share.

Reply:
I go back and forth on this- I like to be able to follow a function from its definition to where its called without changing its name, But it is true that I have made the mistake of calling wrong function by forgetting to pass it though dispatch multiple times... so perhaps I should just conform to your suggestion.

Reply:
I'll change this here and use on in the future but I'm not going to go through all of this PR